src/main/java/io/logz/jmx2graphite/Jmx2GraphiteConfiguration.java

```diff
@@ -2,52 +2,82 @@
 
 import com.typesafe.config.Config;
 
+import java.net.InetAddress;
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.net.UnknownHostException;
 import java.util.concurrent.TimeUnit;
 
 /**
  * @author amesika
  */
 public class Jmx2GraphiteConfiguration {
     /* Jolokia full URL */
-    public String jolokiaUrl;
+    private String jolokiaUrl;
 
-    public String graphiteHostname;
-
-    public int graphitePort;
+    // Graphite server related params
+    private  String graphiteHostname;
+    private int graphitePort;
 
     /* Short name of the sampled service (exposed by the Jolokia URL)", required = false */
```

Comment:
Since we're not Jolokia only implementation, we can delete the part saying Jolokia in the comment